content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
+- view the usage of your enabled PDP products across all projects in that workspace.
+- manage team members and change their roles / permissions in the workspace.
+
+### Project
+
+In each workspace, you can create several projects. A project typically represents an application (a product or service) your organization is offering. You typically have one [Prisma schema](/orm/prisma-schema/) per project.
+
+In each project, you can:
+
+- view and manage all environments in that project.
+
+The number of project you can create in a workspace depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in that workspace:
+
+Here's an overview of the number of projects per workspace on each plan:
+
+|              | **Starter** | **Pro** | **Business** | **Enterprise** |
+| ------------ | ----------- | ------- | ------------ | -------------- |
+| **Projects** | 5           | 10      | 15           | Custom         |
+
+### Environment
+
+An environment is an isolated space to provision PDP products for a given project. Environments typically map to development stages, like `dev`, `staging` or `prod`.
+
+In each environment, you can:
+
+- enable, disable and manage PDP products (Pulse, Accelerate, ...).
+- for **Accelerate**:
+  - set your database connection string.
+  - generate API keys.
+  - change the connection pool size.
+  - enable static IPs.
+- for **Pulse**:
+  - set your database connection string (of a [Pulse-compatible database](/pulse/database-setup)).
+  - configure the _region_ where Pulse is running.
+  - configure database replication.
+  - generate API keys.
+
+The number of environments you can create in a project depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in your workspace:
+
+Here's an overview of the number of environments per project on each plan:
+
+|                  | **Starter** | **Pro**     | **Business** | **Enterprise** |
+| ---------------- | ----------- | ----------- | ------------ | -------------- |
+| **Environments** | 2 / project | 6 / project | 12 / project | Custom         |
+
+## Billing
+
+The [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) you select in your workspace determins how many projects and environments you can create in that workspace:
+
+|                  | **Starter** | **Pro**     | **Business** | **Enterprise** |
+| ---------------- | :---------- | ----------- | ------------ | -------------- |
+| **Projects**     | 5           | 10          | 15           | Custom         |
+| **Environments** | 2 / project | 6 / project | 12 / project | Custom         |
+
+### Per-workspace billing
+
+Billing in PDP is done on a per-workspace basis:
+
+- A pricing plan is selected per workspace. That means, your PDP account can have multiple workspaces where each workspace uses a different pricing plan.
```

Comment:
```suggestion
- A subscription plan is selected per workspace. That means, your user can belong to multiple workspaces where each workspace uses a different plan.
```

Reply:
I can commit this but personally find "pricing plan" clearer than "subscription plan" because:
- "pricing" is also part of the target url and leaves no room for ambiguity misinterpration
-  "subscription" is generally a bit more ambiguous IMO (also in the context of Pulse where we have "subscriptions")
- I _believe_ I/we also used "pricing plan" everywhere else on this page

Reply:
I updated the bullet with your other suggestions though:

- A pricing plan is selected per workspace. That means, a user account can belong to multiple workspaces where each workspace uses a different plan.

Reply:
> I can commit this but personally find "pricing plan" clearer than "subscription plan"

I saw you made the same suggestion elsewhere. Just confirm that we want to go with "subscription plan" and I'll do a find/replace on the page 👍

Reply:
That's how I usually put it :) I have never used pricing plan anywhere. We usually talk about subscriptions to talk about what you have active that is on a specific plan, so I think subscription plan and subscription base price is the right terminology.

Reply:
done with find/replace ✅